rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -787,6 +788,24 @@ void main_window::HandlePupInstallation(QString file_path)
 	Emu.SetForceBoot(true);
 	Emu.Stop();
 
+	struct fonts_manage
+	{
+		main_window* mw;
+
+		fonts_manage(main_window* _mw)
+			: mw(_mw)
+		{
+			// Remove possibly PS3 fonts from database
+			QFontDatabase::removeAllApplicationFonts();
```

Comment:
and call this in gui_application::OnChangeStyleSheetRequest (just move the existing call to the top of the function)

Reply:
Like this? or after the CLI check?